signup/index.php

```diff
@@ -4,8 +4,8 @@
 <head>
 	<title>VITA Appointment Signup</title>
 	<?php require_once "$root/server/header.php" ?>
-	<link rel="stylesheet" href="/assets/css/form.css">
-	<link rel="stylesheet" href="/signup/signup.css">
+	<link rel="stylesheet" href="/dist/assets/css/form.css">
+	<link rel="stylesheet" href="/dist/signup/signup.css">
 	<link rel="stylesheet" href="/assets/css/jquery-ui-datepicker.css">
```

Comment:
I purposefully didn't prepend /dist to this `jquery-ui-datepicker` stuff since it's not ours, I see no reason to run it through our build process, I'm sure it's already been ran through a build process?

Reply:
That's totally fine. In fact, after the restyle, we won't need that file anymore since there is a UNL styled jquery ui datepicker.